_docs/2.0.0/usage/menas-api.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
+title: Usage - Menas API
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+- [Export Entity](#export-entity)
+  - [Example Import Response for Schema](#example-import-response-for-schema)
+- [Entity Import](#entity-import)
+  - [JSON payload](#json-payload)
+  - [Example Export Response for Schema](#example-export-response-for-schema)
+
+
+### Export Entity
+
+This feature exports a single entity from Menas. All exports are versionless, except for connected entities. They are kept as is, but need to be validated before import.
+
+`GET /menas/api/{entity}/exportItem/{name}/{version}`
```

Comment:
How about the fact that schema attachments can also be exported via `/menas/api/schema/export/{name}/{version}`?

Also, maybe this would only complicate things, so I can see why you took this shortcut, just want to know if it was deliberate or not: Even though with our deployments, Menas and its API is usually deployed under `/menas`, technically, it can be operated under a different path-prefix, so maybe one could at least mention it somewhere that this mental shortcut is used in the API docs everywhere.

Reply:
- `/menas` is a mistake it was meant to be something like `{menas_url}/api/...`. Will fix
- I know of export, but it is used for something else and this export does not have any import counterpart. It is not a full entity export but only schema export